src/exercises/exerciseTypes/NextNavigation.js

```diff
@@ -168,60 +178,65 @@ export default function NextNavigation({
             />
             <p>
               <b>
-                {`${matchExerciseProgressionMessage}`} have now moved to your
+                {`${matchExerciseProgressionMessage}`}{" "}
+                {matchWordsProgressCount > 1 ? "have" : "has"} now moved to your
```

Comment:
opportunity to extract utility function? e.g. `pluralize("has", matchWordsProgressCount)`. or is it overkill?